src/plugins/visualizations/public/visualize_app/components/visualize_listing.tsx

```diff
@@ -185,22 +202,19 @@ export const VisualizeListing = () => {
   );
 
   return (
-    <TableListView
+    <TableListView<VisualizeUserContent>
+      id="vis"
       headingId="visualizeListingHeading"
       // we allow users to create visualizations even if they can't save them
       // for data exploration purposes
       createItem={createNewVis}
-      tableCaption={i18n.translate('visualizations.listing.table.listTitle', {
-        defaultMessage: 'Visualize Library',
-      })}
       findItems={fetchItems}
       deleteItems={visualizeCapabilities.delete ? deleteItems : undefined}
       editItem={visualizeCapabilities.save ? editItem : undefined}
```

Comment:
More a question than anything 

https://user-images.githubusercontent.com/20343860/188666581-671e346b-2d45-48af-8e3a-0c2ec5b9a7a1.mp4

I'm testing locally and I'm having issues with the edit button with Lens specific visualizations with sample data...let me know what I'm missing thank you!

Reply:
That's a great catch @rshen91 ! I fixed it in https://github.com/elastic/kibana/pull/140040/commits/8287766207a0eae7e59c0dd60c802e6592a5ffc4